pkg/jx/cmd/opts/common.go

```diff
@@ -1087,6 +1088,16 @@ func (o *CommonOptions) CertManagerClient() (certmngclient.Interface, error) {
 	return o.certManagerClient, nil
 }
 
+// GetNameServers returns the name servers for the clusters domain
+func (o *CommonOptions) GetNameServers() []string {
+	return o.nameServers
+}
+
```

Comment:
Doesn't seem any need for these get/set functions, could you just export the Struct variable instead?

Reply:
True, I was maintaining some function parity...will refactor 😀